src/f32/quat.rs

```diff
@@ -454,8 +454,25 @@ impl Quat {
         self.mul_vec3a(Vec3A::from(other)).into()
     }
 
+    #[inline]
+    /// Adds two quaternions.
+    /// The sum not guaranteed to be normalized.
+    /// NB: Addition is not the same as combining the rotations represented by the two quaternions!
+    /// That corresponds to multiplication.
+    pub fn add_quat(self, other: Self) -> Self {
+        Self(self.0 + other.0)
+    }
+
+    #[inline]
+    /// Multiplies a quaternion with an f32.
+    /// The product is not guaranteed to be normalized.
+    pub fn mul_f32(self, other: f32) -> Self {
+        Self(self.0 * other)
+    }
+
     #[inline]
     /// Multiplies two quaternions.
+    /// If they each represent a rotation, the result will represent the combined rotation.
```

Comment:
I don't remember why I added this function, but regardless your comment update is fine.